app/controllers/games_controller.rb

```diff
@@ -32,19 +34,16 @@ def edit
   # POST /games
   # POST /games.json
   def create
-    @game = Game.new(game_params)
-    @game.current_turn = 0
-    @game.black_player_id = current_player.id if @game.white_player_id == 0
-    
-    respond_to do |format|
+    ActiveRecord::Base.transaction do
```

Comment:
what are you meaning when you say sub transaction? I'm creating the transaction here to make sure the @game and timers are created correctly or none of it gets created and we aren't left with bad db entries.